scripts/copy-azure-teacher.py

```
@@ -0,0 +1,414 @@
+    cursor = local_conn.cursor()
+    # Delete in reverse order to respect FK constraints
+    for config in reversed(TABLES_CONFIG):
+        params = config["delete_params"](teacher_id)
+        try:
+            cursor.execute(config["delete"], params)
+            deleted = cursor.rowcount
+            if deleted > 0:
+                print(f"  Deleted {deleted} rows from {config['table']}")
+        except pyodbc.Error as e:
+            if "Invalid object name" not in str(e):
+                print(f"  Warning: could not delete from {config['table']}: {e}")
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)